src/ingestor.ts

```diff
@@ -182,8 +182,8 @@ export default async function ingestor(req) {
         pin(ipfsBody, process.env.PINEAPPLE_URL),
         issueReceipt(body.sig)
       ]);
-    } catch (e) {
-      capture(e);
+    } catch (e: any) {
+      capture(e.error || e);
```

Comment:
I feel this should be handled inside the capture itself

Reply:
Let me see